code/game/objects/items/clown_items.dm

```diff
@@ -81,6 +102,9 @@
 			to_chat(user, "<span class='notice'>You clean \the [target.name].</span>")
 			target.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
 			target.set_opacity(initial(target.opacity))
+	else if(istype(target, /obj/item/ammo_casing/spent) && bulletclean == TRUE) //No messages otherwise it gives us away
```

Comment:
```suggestion
	else if(bulletclean && istype(target, /obj/item/ammo_casing/spent)) //No messages otherwise it gives us away
```
Take advantage of short circuiting.